### DISCUSSION TITLE
Better UX design

### DISCUSSION THREAD

**Discussion by Ceynou** [FreeLanguageTools/vocabsieve#41] [Ideas]
A way to **easily switch languages** or maybe just a **separate global configuration** that automatically sets the language.
For example you need to set the correct language and everything everytime in configure to import lookups from books of that said language.

I'm not sure this case by case features are a good idea so you can tell me what you think

**Comment by 1over137** [comment]
I've wanted to do that for a while now, but couldn't think of a clean way to implement this. The closest existing feature is setting the VOCABSIEVE_DEBUG variable, which opens a new profile for testing, but it's not easy to do on Windows with installed version.
Ideally it probably should not be fully separated. You might want to share some settings, like Anki note types, but not others, like tags or dictionaries. Would it be good if I store the original values of some settings when switching language, so that it can be restored the next time you switch back to it?

**Reply by Ceynou** [reply]
Storing values is definitely a plus, yes please
